### ISSUE TITLE
How to translate address_form.html in userprofile/snippets

### ISSUE THREAD

**Issue by loongmxbt** [mirumee/saleor#1648]
The `address_edit.html` uses snippets `address_form.html`. This file uses `bootstrap_field` and does not translate it self like other pages via saleor locale. I've found there is an app called `bootstrap4` but can't find the document.

I assume put all the translation in one place(locale) will be better, or provide a address form editing in backend. I hope I was not missing something. Wish some help. Thanks!

```
{% load bootstrap_field from bootstrap4 %}
<div class="i18n-address">
  {% with address_form_lines=address_form.i18n_fields_order %}
    {% if address_form_lines %}
      {% for line in address_form_lines %}
        <div class="row">
          {% for field in line %}
            <div class="col-6">
              {% bootstrap_field field %}
            </div>
          {% endfor %}
        </div>
      {% endfor %}
    {% endif %}
  {% endwith %}
  {% for field in address_form.hidden_fields %}
    {{ field }}
  {% endfor %}
  {% bootstrap_field address_form.country %}
  {% bootstrap_field address_form.phone %}
  <input type="hidden" class="preview" name="preview">
</div>
```

**Comment by elwoodxblues** [comment]
[django-bootstrap4](http://django-bootstrap.readthedocs.io/) is an external package that we use for rendering forms. Address forms live in `account/18n.py` file and there you'll find all the labels. All of them are marked for translations so the whole form should be properly translated when changing the language.

**Comment by opticrew** [comment]
Excuse me, I begin with Saleor, can you explain me the different steps ?
 If I understand, I have to customize the different AREA_TYPE_TRANSLATIONS and AUTOCOMPLETE_MAPPING in the "account/18n.py" like for example : 

('first_name', 'prenom'),
('last_name', 'nom'),
...

**Comment by Pacu2** [comment]
Resolved in #1940

**Comment by Pacu2** [comment]
@opticrew Not sure if we are on the same page, but why do you want to change autocomplete mapping?
Saleor comes with full French translation as far as I'm concerned,  translations are managed via [transifex](https://www.transifex.com/mirumee/saleor-1/)
